app/user/rest.py

```diff
@@ -569,7 +569,7 @@ def send_user_reset_password():
         notification_type=template.template_type,
         api_key_id=None,
         key_type=KEY_TYPE_NORMAL,
-        reply_to_text=service.get_default_reply_to_email_address(),
+        reply_to_text=None,
     )
```

Comment:
Ditto. `reply_to_text` is not necessary.

Reply:
deleted